### PR TITLE
Haveno Performance Report

### PR DESCRIPTION
**Pull request by agg505**

### CODE REVIEW

**Comment by erciccione** [comment]
ping @agg505

**Comment by erciccione** [comment]
Ping @agg505.

**Comment by agg505** [comment]
@erciccione Fix done, please have a check. And I apologize for no response so long because I just monitored this thread(https://github.com/haveno-dex/haveno/issues/20) instead of this one.

File: haveno-performance-report.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+|                                |JFR Periodic Tasks                 |2,748 ms (10%)      |Ignore instrument thread|
+|                                |main                                 |2,731 ms (10%)      |Not found|
+
+# Flame Graphs for Some Thread Types
+
+Here are some flame graphs for each type of threads which are neither JavaFX nor instrument related above.
+
+## RMI TCP Connection(*)-127.0.0.1
+
+![rmi tcp flame graph](media/performance-report/RMI TCP Connection(*)-127.0.0.1.jpg)
```
Comment (erciccione) [inline haveno-performance-report.md:117]: This is not a valid link. Probably better to use a simpler filename, like `rmi-tcp-conn-127.jpg` or something like that.

File: haveno-performance-report.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+## InvokeLaterDispatcher
+
+![InvokeLaterDispatcher](media/performance-report/InvokeLaterDispatcher.jpg)
+
+## RMI TCP Connection(idle)
+
+![RMI TCP Connection(idle)](media/performance-report/RMI TCP Connection(idle).jpg)
```
Comment (erciccione) [inline haveno-performance-report.md:125]: This is not a valid link. Probably better to use a simpler filename, like `rmi-tcp-conn-idle.jpg` or something like that.

**Review by erciccione — CHANGES_REQUESTED** [review]
No worries. There are 2 broken links, we can merge after those are fixed.

**Comment by agg505** [comment]
@erciccione The files now have a better name.

**Review by erciccione — APPROVED** [review]

**Pull request merged**